learn/src/main/java/org/navgurukul/learn/courses/repository/LearnRepo.kt

```diff
@@ -209,11 +209,15 @@ class LearnRepo(
     }
 
     suspend fun getCompletedContentsIds(courseId: String): Flow<CompletedContentsIds> {
-        val contentList = courseApi.getCompletedContentsIds(courseId)
-
-        updateCompletedContentInDb(contentList)
-
-        return flow { emit(contentList) }
+        return flow {
+            try {
```

Comment:
@Dhan-shri This try catch be should be at the start of this function and not inside flow. 
Is there any specific benefit of doing this?